chainer/links/connection/convolution_2d.py

```diff
@@ -55,8 +55,10 @@ def __init__(self, in_channels, out_channels, ksize, stride=1, pad=0,
         self.pad = _pair(pad)
         self.use_cudnn = use_cudnn
         self.out_channels = out_channels
-        self.initialW = initialW
-        self.wscale = wscale
+        # For backward compatibility, the scale of weights is proportional to
+        # the square root of wscale.
+        self.W_initializer = initializers._get_initializer(
+            initialW, scale=math.sqrt(wscale))
```

Comment:
The same as above; for the users who use the member variables ``initialW`` and ``wscale``, it breaks backward compatibility. I think this PR should be merged at the next major release.